### DISCUSSION TITLE
Weblate

### DISCUSSION THREAD

**Discussion by vitalitk** [RikkaApps/Shizuku#219] [Q&A] [answered]
Hello. It seems that your Weblate SSH client has expired. Chrome gives a NET::ERR_CERT_DATE_INVALID error. The SSH certificate expired on June 8th.

**Comment by RikkaW** [comment, marked as answer]
Fixed now :D

**Reply by vitalitk** [reply]
Thanks

**Comment by Coool** [comment]
What's happening with Weblate[^weblate]? Recently was relesed 13.6.0[^new_release] version and without merging Latvian[^latvian_language] contribution made on Weblate. Can somebody give explanation?

[^weblate]: [Weblate](https://weblate.rikka.app)
[^latvian_language]: #1020
[^new_release]: [v13.6.0.r1086.2650830c](https://github.com/RikkaApps/Shizuku/releases/tag/v13.6.0)
